src/snowflake/cli/_plugins/stage/diff.py

```diff
@@ -83,18 +83,27 @@ def enumerate_files(path: Path) -> List[Path]:
     return paths
 
 
-def strip_stage_name(path: str) -> StagePathType:
-    """Returns the given stage path without the stage name as the first part."""
-    return StagePathType(*path.split("/")[1:])
+def relative_to_stage_path(path: str, stage_subirectory: str) -> StagePathType:
+    """
+    @param path: file path on the stage.
+    @param stage_subirectory: subdirectory of stage.
+    @return: path of file relative to the stage_path
+    """
+    wo_stage_name = StagePathType(*path.split("/")[1:])
+    relative_path = str(wo_stage_name).removeprefix(stage_subirectory).lstrip("/")
```

Comment:
can't tell what these are doing. I see the first command removing first part left of /, and the second line doing something similar.

- Adding comments might help.
- Curious if Path can somehow be leveraged to calculate relative path instead?

Reply:
Added comment and updated method. 
Can't use Path at all since stage name can be a quoted identifier which can have any character including a "/" in its name